README.md

```diff
@@ -311,33 +310,11 @@ The `mcpadm` scripts can also get, deploy and destroy terraform infrastructure,
 management of multiple service versions.
 
 ## Anchors
-For example, [YAML anchors](https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases) can be used to reference values from 
-other fields. For example, if you wanted `service` to be the same as name, would write:
-
-```yaml
-name: &name ecat-admin
-service: *name
-```
-
-Reused anchors overwrite previous values. I.e. when anchors are repeated, the value of the last found anchor will be
-used. For example, with the configuration:
-
-```yaml
-components:
-  common:
-    env_variables:
-      'env': dev
-    threadsafe: True
-    name: &name common-name
-  specs:
-    app1:
-      name: &name ecat-admin
-      runtime: custom
-      env: flex
-      service: *name
-```
+We recommend using [YAML anchors](https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases) to reduce duplication and ensure
+consistency between various configs. Description of this feature is available in a variety of external sources,
+e.g. [simple summary lives here](https://www.educative.io/blog/advanced-yaml-syntax-cheatsheet#anchors). 
```

Comment:
This is information about a widely documented format. In my opinion, it shouldn't be our job to try to re-explain fundamental concepts, so am proposing to reduce the amount of docs we maintain in this manner.